scripts/release.py

```diff
@@ -299,6 +302,7 @@ def create_notebook(dd_repo, name, rn, base, rc, patch):
                     rn_piece = re.sub("  ", " ", rn_piece)
                     rn_piece = re.sub("``", "`", rn_piece)
                 except Exception:
```

Comment:
Given how large the `try` block is, I strongly recommend being specific about the exception types that this `except` statement intends to catch. Currently the code does not make clear which pieces of the `try` block can fail and how.